pattern-library/partials/_mode-bar.html

```diff
@@ -62,6 +62,51 @@
         </ul>
 
         <ul class="account-menu">
+            <li>
+                <a href="#" data-modal="collections">
+                    <svg class="iconic">
```

Comment:
Could you try using tabindex here?

Reply:
Oh, ok. Here too. Will do so too Thanks.